src/models/countries.js

```diff
@@ -0,0 +1,101 @@
+export default (sequelize, DataTypes) => {
+  const Country = sequelize.define("Country", {
+    nameOfCountry: {
+      type: DataTypes.ENUM,
+      values: [
+        "Nigeria",
+        "Ethiopia",
+        "Egypt",
+        "Democratic Republic of the Congo",
+        "Tanzania",
+        "South Africa",
+        "Kenya",
+        "Uganda",
+        "Algeria",
+        "Sudan",
+        "Morocco",
+        "Mozambique",
+        "Ghana",
+        "Angola",
+        "Somalia",
+        "Ivory Coast",
+        "Madagascar",
+        "Cameroon",
+        "Burkina Faso",
+        "Niger",
+        "Malawi",
+        "Zambia",
+        "Mali",
+        "Senegal",
+        "Zimbabwe",
+        "Chad",
+        "Tunisia",
+        "Guinea",
+        "Rwanda",
+        "Benin",
+        "Burundi",
+        "South Sudan",
+        "Eritrea",
+        "Sierra Leone",
+        "Togo",
+        "Libya",
+        "Central African Republic",
+        "Mauritania",
+        "Republic of the Congo",
+        "Liberia",
+        "Namibia",
+        "Botswana",
+        "Lesotho",
+        "Gambia",
+        "Gabon",
+        "Guinea-Bissau",
+        "Mauritius",
+        "Equatorial Guinea",
+        "Eswatini",
+        "Djibouti",
+        "Réunion (France)",
+        "Comoros",
+        "Cape Verde",
+        "Mayotte (France)",
+        "São Tomé and Príncipe",
+        "Seychelles",
+      ],
+      allowNull: false,
+    },
+    gallery: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    capital: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    population: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    officialLanguage: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    region: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    currency: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  });
+
+  Country.associate = (models) => {
+    Country.hasMany(models.Culture, {
+      as: "cultureCountry",
+      foreignKey: "countryId",
+      onDelete: "cascade",
+      hooks: true,
```

Comment:
remove this two lines of code 
```
onDelete: "cascade",
      hooks: true,
```

its not needed for now

Reply:
done